feature-libs/asm/styles/components/_customer-selection.component.scss

```diff
@@ -13,6 +13,10 @@
       margin: 0 0 15px;
       min-width: auto;
 
+      cx-form-errors p {
```

Comment:
please add feature toggle to avoid break change.

Reply:
Done